### DISCUSSION TITLE
Does Skeleton support tailwind prefix ?

### DISCUSSION THREAD

**Discussion by dror-weiss** [skeletonlabs/skeleton#2345] [Q&A] [answered]
My tailwind prefix is set to "tw-"; how can I configure Skeleton to prefix all tailwind classes it uses with that?

**Comment by endigo9740** [comment, marked as answer]
@dror-weiss unfortunately prefixes are kind of "all or nothing". If you set a prefix in your Tailwind config it will expect ALL classes defining the Skeleton features to also have that prefix - but they won't. This means this will not work.

So generally speaking prefix is not supported in Skeleton, even though we would love for it to be!

**Reply by dror-weiss** [reply]
@endigo9740 , thanks for the explanation.
Is [this solution](https://github.com/saadeghi/daisyui/blob/master/src/lib/addPrefix.js) from [daisy-ui](https://daisyui.com/docs/config/#prefix) can be of any help to you?

**Reply by endigo9740** [reply]
Interesting, we'll take a look. Thanks for sharing!
